.github/workflows/test_build_wheels_windows_without_cuda.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Build Windows Wheels without CUDA
+
+on:
+  pull_request:
+    paths:
+      - .github/actions/setup-binary-builds/action.yml
+      - .github/workflows/test_build_wheels_windows_without_cuda.yml
+      - .github/workflows/build_wheels_windows.yml
+      - .github/workflows/generate_binary_build_matrix.yml
+      - tools/scripts/generate_binary_build_matrix.py
```

Comment:
Why don't we want to test the same on `push`?

Reply:
I don't know. I followed what other workflows do. Maybe we should, but not in this PR.